src/main/client/src/reducers/table/setActivePage.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import { SET_ACTIVE_PAGE, CLOSE_LAYER } from '../../constants/actionTypes';
+
+const initialState = {
+    page: [],
+    arrIndex: {},
+}
+
+type Action = {
+    type: string,
+    page: Object[],
+};
+
+export default (state: State = initialState, action: Action) => {
+    switch (action.type) {
+        case SET_ACTIVE_PAGE:
+            var index = initialState.arrIndex[action.page.layerId];
```

Comment:
Pitäisikö käyttä let eikä var.
```suggestion
            let index = initialState.arrIndex[action.page.layerId];
```